### PR TITLE
update goldens after mixer staging push

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/integration_tests/test_data/detection_api_bio/whatisthephylumofvolvox/debug_info.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "entities_detected": [
     "volvox"
   ],
-  "entities_resolved": "(name: Volvox, dcid: bio/D044446); (name: Volvox, dcid: dc/003ye83smktt7); (name: Volvox, dcid: dc/3rbv0rhm76sg8); (name: Volvox, dcid: dc/5zhg2q9l2n378); (name: Volvox, dcid: dc/953xfs4xh2bm5); (name: Volvox, dcid: dc/9fzm6f7k7etm4); (name: Volvox, dcid: dc/9xy7ymdmzyzdf); (name: Volvox, dcid: dc/b090kbyb0sz5b); (name: Volvox, dcid: dc/ccyhttectv5n2); (name: Volvox, dcid: dc/cd735b0hd6nh8); (name: Volvox, dcid: dc/ds26y3mt8z3c4); (name: Volvox, dcid: dc/erp74w62y7x4h); (name: Volvox, dcid: dc/fnqkx4gtrh1f7); (name: Volvox, dcid: dc/fzfq3lff3kf34); (name: Volvox, dcid: dc/ghtpcec74knl7); (name: Volvox, dcid: dc/h8ncxq8y6bkj2); (name: Volvox, dcid: dc/pdhk5jc41d636); (name: Volvox, dcid: dc/vxzgmlszw0zfh); (name: Volvox, dcid: dc/xvhwj1943wgf9); (name: Volvox, dcid: dc/yg1kw13e813e9); (name: Volvox, dcid: dc/z9x1mlh4pxzb7); (name: Volvox, dcid: dc/zsn19vpl9fy76); ",
+  "entities_resolved": "(name: Volvox, dcid: bio/D044446); ",
```
Comment (pradh) [inline server/integration_tests/test_data/detection_api_bio/whatisthephylumofvolvox/debug_info.json:9]: expected?

File: server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json [hunk 2]
```diff
@@ -292,60 +292,6 @@
...
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/SI_POV_EMP1.AGE--Y15T24",
-                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--F",
-                      "sdg/SI_POV_EMP1.AGE--Y15T24__SEX--M"
-                    ],
-                    "title": "Employed Population 15 to 24 Years Old Below International Poverty Line, by Sex (%) in India",
```
Comment (pradh) [inline server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: Wonder if this is from SDG refresh or some NL index change.

**Review by pradh — APPROVED** [review]
Thanks!

File: server/integration_tests/test_data/detection_api_bio/whatisthephylumofvolvox/debug_info.json (hunk 1 above)
Comment (chejennifer) [reply server/integration_tests/test_data/detection_api_bio/whatisthephylumofvolvox/debug_info.json:9]: yes, expected

File: server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json (hunk 2 above)
Comment (chejennifer) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: hm, should not be from NL index change if there isn't any index changes in this PR?

Comment (pradh) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: Ah, good point.  @n-h-diaz or @dwnoble - could you confirm please?

Comment (n-h-diaz) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: There's still data in the refresh for these: https://autopush.datacommons.org/tools/statvar#sv=sdg/SI_POV_EMP1.AGE--Y15T24__SEX--F

Comment (n-h-diaz) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: It doesn't look like there's any data in the refresh for Indian states for these variables

Comment (chejennifer) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: so diffs are ok then?

Comment (dwnoble) [reply server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: Yes looks like the data was removed with this refresh. I think it's OK to proceed, and ill bring this up with UN separately

**Review by dwnoble — APPROVED** [review]

**Comment by chejennifer** [comment]
thanks for the reviews!

**Pull request merged**